easybuild/easyconfigs/e/Eclipse/Eclipse-4.8-parallel.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'Tarball'
+
+name = 'Eclipse'
+version = '4.8'
+versionsuffix = "-parallel-photon-R"  # For scientific computing
+
+homepage = 'http://www.eclipse.org'
+description = """Integrated Development Environment"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = [
+    'http://www.eclipse.org/downloads/download.php?'
+    'file=/technology/epp/downloads/release/photon/R/'
```

Comment:
@SimonPinches This is short enough to fit in < 120 chars on a single ine?

Reply:
Okay - I'll push a fix for this.